sympy/printing/preview.py

```diff
@@ -183,8 +186,9 @@ def preview(expr, output='png', viewer=None, euler=True, packages=(),
     try:
         workdir = tempfile.mkdtemp()
 
-        with open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
-            fh.write(latex_main % latex_string)
+        with io.open(join(workdir, 'texput.tex'), 'w', encoding='utf-8') as fh:
+            rendered = latex_main % latex_string
+            fh.write(u(rendered.replace(r'\u', r'\\u')))
```

Comment:
I have just tested the branch and it doesn't seem to work. I get the same error message as before. 

here is my test example:

```py
import os
os.environ['SYMPY_DEBUG'] = 'True'
from IPython.display import display, Latex
from sympy.interactive import printing
printing.init_printing(use_latex='png', forecolor='Red')
from sympy import *
params = symbols('k l m'.split())
display(params)
```